src/doc/help.py

```diff
@@ -21,6 +21,8 @@
  ***************************************************************************/
 """
 
+#modifié le 30/05/2017 par GOREE Rachel
+
```

Comment:
Can we remove these two lines? If you care about attributions, we can add in the header of this file line 10. Github will also track contributors to the project on this page: https://github.com/Gustry/GeoHealth/graphs/contributors your name will be added when this pull request will be merged.